Automattic/jetpack
# VideoPress Block: Add setting for playing video inline

We added support to the VideoPress player to support the `playsinline` attribute which will prevent a video from automatically playing full screen when on mobile. This PR adds a user-facing UI for that attribute to be set.

<img width="279" alt="Screen Shot 2021-03-29 at 2 01 09 PM" src="https://user-images.githubusercontent.com/789137/112893424-e61c2e00-9097-11eb-8238-4720397d15cd.png">

#### Changes proposed in this Pull Request:
* Adds a new toggle to the VideoPress block settings (disabled by default) that will allow a user to set if the video should be played inline in the browser.

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* Before applying PR, add a VideoPress block to a post with a video and save it.
* Apply the PR and rebuild Jetpack. Load up the post you saved to ensure the block still loads properly. The `Plays Inline` setting should be untoggled.
* Add another new post and add a video block. Turn on the `Plays Inline` toggle, and publish the post. 
* Inspect the video, and you should see the `playsinline` attribute applied to the `video` tag.
* Bring up the url on a mobile device (I tested on iPhone 12). Pressing play should play the video inline on the page, not taking you full screen.
